cmd/katapult/data_centers_test.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/krystal/go-katapult"
+	"github.com/krystal/go-katapult/core"
+	"github.com/stretchr/testify/assert"
+)
+
+var dcs = []*core.DataCenter{
+	{
+		ID:        "POG1-ID",
+		Name:      "hello",
+		Permalink: "POG1",
+		Country: &core.Country{
+			ID:   "POG",
+			Name: "Pogland",
+		},
+	},
+	{
+		ID:        "GB1-ID",
+		Name:      "hello",
+		Permalink: "GB1",
+		Country: &core.Country{
+			ID:   "UK",
+			Name: "United Kingdom",
+		},
+	},
+}
+
+type mockDataCentersClient struct {
+	dcs    []*core.DataCenter
+	throws string
+}
+
+func (m mockDataCentersClient) List(context.Context) ([]*core.DataCenter, *katapult.Response, error) {
+	if m.throws != "" {
+		return nil, nil, errors.New(m.throws)
+	}
+	return m.dcs, nil, nil
+}
+
+func (m mockDataCentersClient) Get(
+	_ context.Context, ref core.DataCenterRef) (*core.DataCenter, *katapult.Response, error) {
+	if m.throws != "" {
+		return nil, nil, errors.New(m.throws)
+	}
+	for _, v := range m.dcs {
+		if v.ID == ref.Permalink {
```

Comment:
I think this should compare against `v.Permalink`, not `v.ID`. Additionally the IDs in the data center fixtures should be `dc_` followed by a 16 character long random alphanumeric string, for example: `dc_9UVoPiUQoI1cqtR0`